ecosante/newsletter/blueprint.py

```diff
@@ -194,7 +194,8 @@ def test():
                 ],
                 recommandations
             ),
-            recommandations=recommandations
+            recommandations=recommandations,
+            indice_uv=int(request.form.get("indice_uv"))
```

Comment:
C’est plus pour te montrer que ça existe, c’est pas bien grave
```suggestion
            indice_uv=request.form.get("indice_uv", type=int)
```